packages/opentelemetry-exporter-collector-grpc/src/util.ts

```diff
@@ -67,7 +67,8 @@ export function onInit<ExportItem, ServiceRequest>(
           collector.send(item.objects, item.onSuccess, item.onError);
         });
       }
-    });
+    })
+    .catch(err => collector.logger.error(err.message));
```

Comment:
If there is an error during onInit it will now be masked by this. Was that intended?

Reply:
nope! adding the lint skip here